index.js

```diff
@@ -319,7 +336,15 @@ class SafeView extends Component {
         return statusBarHeight(isLandscape);
       }
       case 'bottom': {
-        return isIPhoneX ? (isLandscape ? 24 : 34) : 0;
+        if (isIPhoneX) {
+          return isLandscape ? 24 : 34;
+        }
+
+        if (isNewIPadPro) {
+          return 20;
```

Comment:
Why `20` for iPadPro and not `34` to be the same as the iPhoneX? They both have the "Home Line" and need the same amount of clearance right? @cesargdm

Reply:
iPhoneX
![Screen Shot 2020-04-13 at 10 00 25](https://user-images.githubusercontent.com/1085976/79082884-a583b180-7d6d-11ea-95b4-b8665541c12c.png)

iPadPro
![Screen Shot 2020-04-13 at 10 00 31](https://user-images.githubusercontent.com/1085976/79082881-a3b9ee00-7d6d-11ea-8fa3-f269562158ca.png)